src/Styleguide/Components/Lightbox/CloseButton.tsx

```diff
@@ -0,0 +1,23 @@
+import { color, space } from "@artsy/palette"
+import React from "react"
+import styled from "styled-components"
+
+const CloseButtonContainer = styled.svg`
+  width: ${space(6)}px;
+  height: ${space(6)}px;
+  padding: ${space(3) / 2}px;
```

Comment:
This is a little strange from a spacing perspective. I'm technically using invalid space units here, but I'm doing it for a good reason. Normally this button would be 3 units wide and 3 units tall with 3 units of margin. That's fine, but it makes the click area a bit small. This way makes it take up the same amount of space but opens up the click area to have a little more room. Definitely nice on mobile.